azure/functions/decorators/dapr.py

```diff
@@ -0,0 +1,170 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think it would be better if we move dapr and dapr_function_app in its own directory inside decorators?